index.html

```diff
@@ -557,24 +566,66 @@ <h3>Terminology</h3>
 
           <dt><dfn class="export" data-lt="controller(s)|Controllers">controller</dfn></dt>
           <dd>
-An entity that is [=authorized=] to perform an action with a specific resource,
-such as update a [=controller document=] or use a cryptographic key to generate
-a digital signature.
+            <p>
+An entity that is capable of performing an action with a specific resource,
+such as updating a [=controller document=] or generating a [=proof=] using a [=verification method=].
+            </p>
           </dd>
 
           <dt><dfn class="export">controller document</dfn></dt>
           <dd>
-A set of data that specifies one or more relationships between a
-[=controller=] and a set of data, such as a set of public cryptographic keys.
+            <p>
+A document that contains cryptographic material and service
+endpoints for the purposes of verifying proofs from, and interacting
+with, the [=controller=] of an identifier.
+            </p>
           </dd>
 
           <dt><dfn data-lt="subjects">subject</dfn></dt>
           <dd>
-The entity identified by the `id` property in a [=controller document=].
-Anything can be a subject: person, group, organization, physical thing, digital
-thing, logical thing, etc.
+            <p>
+An entity that is referred to by the `id` property in a [=controller document=]
+when it is used as a subject in other contexts, such as authentication or
+attestations, e.g., Verifiable Credentials. Anything can be a subject: 
+person, group, organization, physical thing, digital thing, logical thing, etc. 
+            </p>
+            <p class="note">
+It is expected that the subject referred to by the id of a controller document 
+be consistent over time, so that a set of credentials with a given identifer as 
+subject can be taken to be about the same entity. However, it *is* possible to 
+issue credentials with a subject identifier _without_ the subject even being 
+aware of it.
+
+It is expected that credential issuers will ask the subject for identifiers 
+and that those subjects would demonstrate proof of control before the issuer
+issues the credential. However, there are valid cases where issuers will 
+use identifiers that are not provably under the control of the subject. 
+
+However, even in cases
+where the subject of the ID does prove control, the interpretation of the subject
+remains contextual and potentially ambiguous.
```

Comment:
We need to stem the bleeding somehow.

@David-Chadwick Has been wrestling with this issue from the very beginning and he's not the only one.

In centralized systems people believe they can trust identifiers because there is some authority you can appeal to. But in a decentralized system, developers need to understand they don't have an authority to bail them out. What we do have have is cryptography and time. From that, we can build a usable intersubjectivity.

The other alternative would be to get rid of Controller documents altogether an return them to their decentralized form where these notions are primary and understandable. There's no reason the VCDM can't reference the W3C published standard.

Reply:
```suggestion
It is expected that credential issuers will ask the controller for identifiers 
and that those controllers would demonstrate proof of control before the issuer
issues the credential. However, there are valid cases where issuers will 
use identifiers that are not provably under the control of the controller.

However, even in cases
where the controller of the ID does prove control, the interpretation of the subject
remains contextual and potentially ambiguous.
```

Reply:
In this case, the operative issue is the 'subject' proving control. Not the controller.

The controller, by definition, is someone who can prove control.

The problem is that DIDs can be used as subjects without the controller being involved at all.

So, the expectation is that issues will ask the subject to prove control. They may not. For good reasons or bad.

But only if the subject does prove control do we have any reason to expect the controller == subject.